Marsiple/App/Post Details/PostDetailsViewController.swift

```diff
@@ -48,15 +48,23 @@ extension PostDetailsViewController: UITableViewDataSource {
     }
 }
 
+private extension PostDetailsViewController {
+    @objc func commentButtonTapped() {
+        let commentView = CommentViewController()
+        navigationController?.pushViewController(commentView, animated: true)
```

Comment:
I'd say that it's okay to do `navigationController?.pushViewController(CommentViewController(), animated: true)` here, as the statement is very compact.